src/fr/adrienbrault/idea/symfony2plugin/routing/RouteHelper.java

```diff
@@ -85,6 +85,9 @@
             }
         }
 
+        // _fragment was introduced in Symfony 3.3 do we need a project version check?
```

Comment:
as version check is just an if condition. whats about to add to it simply? `fr.adrienbrault.idea.symfony2plugin.util.SymfonyUtil#isVersionLessThenEquals`

Reply:
Great, didnt know the methods exists, will amend.

Reply:
Done.